src/platform/forms-system/src/js/review/submit-states/GenericError.jsx

```diff
@@ -3,28 +3,15 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 // components
-import ErrorMessage from 'platform/forms/components/common/alerts/ErrorMessage';
 import PreSubmitSection from 'platform/forms/components/review/PreSubmitSection';
 import FormSaveErrorMessage from 'platform/forms/components/review/FormSaveErrorMessage';
 import { Column, Row } from 'platform/forms/components/common/grid';
 // const FormSaveErrorMessage = props => props?.children;
 export default function GenericError(props) {
-  const { appType, formConfig, renderErrorMessage, onSubmit, testId } = props;
+  const { formConfig, onSubmit, testId } = props;
   let submitButton;
-  let submitMessage;
 
-  if (renderErrorMessage) {
-    submitMessage = <FormSaveErrorMessage formConfig={formConfig} />;
-  } else {
-    submitMessage = (
-      <ErrorMessage
-        active
-        title={`We’re sorry, the ${appType} didn’t go through.`}
-        message="You’ll have to start over. We suggest you wait 1 day while we fix
-        this problem."
-      />
-    );
-  }
```

Comment:
Since we are removing `renderErrorMessage` because it will always be true (due to adding it to the `SubmitController` in `ReviewPage`), we can remove the else branch here. This will effect some of the unit tests, since they were using some components in ways where `renderErrorMessage` was falsey.

Reply:
I think I like the idea of having this if/else use a condition based on `formConfig.saveInProgress.enabled`. At least that helps us consolidate all of the SiP control into the same place, rather than spread out across components/routes/props/etc.